ingestion-data/production/collections/nldas2.json

```diff
@@ -0,0 +1,125 @@
+{
+    "id": "nldas2",
+    "type": "Collection",
+    "title": "NLDAS2 precipitation",
+    "links": [],
+    "description": "NLDAS-2 provides a fine-scale (1 km) meteorological forcing (precipitation) in both retrospective and near real-time over North and Central America, including Alaska, Hawaii, and Puerto Rico, by leveraging high-quality gauge, satellite, and model datasets through advanced data assimilation methods. Read more: https://ldas.gsfc.nasa.gov/nldas/v2",
+    "extent": {
+        "spatial": {
+            "bbox": [
+                [
+                    -41.2,
+                    -82.6,
+                    9.8,
+                    -59
+                ]
+            ]
+        },
+        "temporal": {
+            "interval": [
+                [
+                    "2001-01-01T00:00:00Z",
+                    "2023-06-01T00:00:00Z"
+                ]
+            ]
+        }
+    },
+    "license": "MIT",
+    "stac_extensions": [
+        "https://stac-extensions.github.io/render/v1.0.0/schema.json",
+        "https://stac-extensions.github.io/item-assets/v1.0.0/schema.json"
+    ],
+    "assets": {
+        "thumbnail": {
+          "href": "https://thumbnails.openveda.cloud/nldas-cover.png",
+          "type": "image/jpeg",
+          "roles": ["thumbnail"],
+          "title": "Thumbnail",
+          "description": "Photo by [LDAS-NASA](https://eoimages.gsfc.nasa.gov/images/imagerecords/151000/151897/mississippi_oli_2023259_lrg.jpg) (Landsat 8 — OLI image of the Mississippi River below Memphis, Tennessee on September 16, 2023 at near record low water levels, limiting barge shipments, threatening drinking water supplies, agriculture, and ecosystems.)"
+        }
+    },
+    "item_assets": {
+        "cog_default": {
+            "type": "image/tiff; application=geotiff; profile=cloud-optimized",
+            "roles": [
+                "data",
+                "layer"
+            ],
+            "title": "Default COG Layer",
+            "description": "Cloud optimized default layer to display on map"
+        }
+    },
+    "dashboard:is_periodic": false,
+    "dashboard:time_density": "month",
+    "stac_version": "1.0.0",
+    "renders": {
+        "dashboard": {
+            "colormap": {
```

Comment:
in the veda-config dashboard preview, the collections are rendered with magma

```
    sourceParams:
      colormap_name: magma
      rescale:
        - 0
        - 200
      nodata: 0
```
https://github.com/NASA-IMPACT/veda-config/pull/441/files#diff-3fd4b26d4ca66c030eec51a4fc0506e4e1f6fe620649d619bcb2ee43df3e62a7R30

Reply:
@anayeaye does this look good?
```
{
    "id": "nldas2",
    "type": "Collection",
    "title": "NLDAS-2 Precipitation Forcing Dataset",
    "links": [],
    "description": "NLDAS-2 provides a fine-scale (1 km) meteorological forcing (precipitation) in both retrospective and near real-time over North and Central America, including Alaska, Hawaii, and Puerto Rico, by leveraging high-quality gauge, satellite, and model datasets through advanced data assimilation methods. Read more: https://ldas.gsfc.nasa.gov/nldas/v2",
    "extent": {
        "spatial": {
            "bbox": [
                [
                    -41.2,
                    -82.6,
                    9.8,
                    -59
                ]
            ]
        },
        "temporal": {
            "interval": [
                [
                    "2001-01-01T00:00:00Z",
                    "2023-06-01T00:00:00Z"
                ]
            ]
        }
    },
    "license": "MIT",
    "stac_extensions": [
        "https://stac-extensions.github.io/render/v1.0.0/schema.json",
        "https://stac-extensions.github.io/item-assets/v1.0.0/schema.json"
    ],
    "assets": {
        "thumbnail": {
            "href": "https://thumbnails.openveda.cloud/nldas-cover.png",
            "type": "image/jpeg",
            "roles": [ "thumbnail" ],
            "title": "Thumbnail",
            "description": "Photo by [LDAS-NASA](https://eoimages.gsfc.nasa.gov/images/imagerecords/151000/151897/mississippi_oli_2023259_lrg.jpg) (Landsat 8 — OLI image of the Mississippi River below Memphis, Tennessee on September 16, 2023 at near record low water levels, limiting barge shipments, threatening drinking water supplies, agriculture, and ecosystems.)"
        }
    },
    "item_assets": {
        "cog_default": {
            "type": "image/tiff; application=geotiff; profile=cloud-optimized",
            "roles": [
                "data",
                "layer"
            ],
            "title": "Default COG Layer",
            "description": "Cloud optimized default layer to display on map"
        }
    },
    "dashboard:is_periodic": false,
    "dashboard:time_density": "month",
    "stac_version": "1.0.0",
    "renders": {
        "dashboard": {
            "assets": [
                "cog_default"
            ],
            "colormap_name": "magma",
            "rescale": [
                [
                    0,
                    200
                ]
            ]
        }
    },
    "providers": [
        {
            "name": "NLDAS",
            "url": "https://ldas.gsfc.nasa.gov/nldas",
            "roles": [
                "producer",
                "processor",
                "licensor"
            ]
        },
        {
            "name": "NASA VEDA",
            "url": "https://www.earthdata.nasa.gov/dashboard/",
            "roles": [
                "host"
            ]
        }
    ]
}
```

Reply:
@slesaad I'm comparing to a collection in prod and maybe also bidx is needed? Plus a we're using a pretty default title it looks like. But _think_ bidx is not needed and the title is certainly optional

```
curl -X 'GET' \
  'https://openveda.cloud/api/stac/collections/nightlights-hd-1band' \
  -H 'accept: application/json' | jq .renders

{
  "dashboard": {
    "bidx": [
      1
    ],
    "title": "VEDA Dashboard Render Parameters",
    "assets": [
      "cog_default"
    ],
    "rescale": [
      [
        0,
        255
      ]
    ],
    "colormap_name": "inferno"
  }
}
```